backend/molgenis-emx2/src/main/java/org/molgenis/emx2/SelectColumn.java

```diff
@@ -9,10 +11,12 @@ public class SelectColumn {
   private Map<String, SelectColumn> children = new LinkedHashMap<>();
   private int limit = 0;
   private int offset = 0;
+  private Filter filter;
   private Map<String, Order> orderBy = new LinkedHashMap<>();
 
   public SelectColumn(String column) {
     this.column = column;
+    this.filter = new FilterBean(AND);
```

Comment:
So only and queries are possible within the nested filter?

Reply:
no. This is just the default.